[2023-07-09T08:31:09.232+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.apply_pca manual__2023-07-09T08:30:57.976741+00:00 [queued]>
[2023-07-09T08:31:09.266+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.apply_pca manual__2023-07-09T08:30:57.976741+00:00 [queued]>
[2023-07-09T08:31:09.269+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-09T08:31:09.319+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): apply_pca> on 2023-07-09 08:30:57.976741+00:00
[2023-07-09T08:31:09.328+0000] {standard_task_runner.py:57} INFO - Started process 2561 to run task
[2023-07-09T08:31:09.333+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'apply_pca', 'manual__2023-07-09T08:30:57.976741+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpa7fo2r3y']
[2023-07-09T08:31:09.334+0000] {standard_task_runner.py:85} INFO - Job 97: Subtask apply_pca
[2023-07-09T08:31:09.450+0000] {task_command.py:410} INFO - Running <TaskInstance: etl_dag.apply_pca manual__2023-07-09T08:30:57.976741+00:00 [running]> on host 5865fe7b5e5b
[2023-07-09T08:31:09.852+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='apply_pca' AIRFLOW_CTX_EXECUTION_DATE='2023-07-09T08:30:57.976741+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-09T08:30:57.976741+00:00'
[2023-07-09T08:31:09.877+0000] {warnings.py:110} WARNING - /opt/***/dags/test.py:68: FutureWarning: Dropping of nuisance columns in DataFrame reductions (with 'numeric_only=None') is deprecated; in a future version this will raise TypeError.  Select only valid columns before calling the reduction.
  df.fillna(df.mean(), inplace=True)

[2023-07-09T08:31:09.884+0000] {warnings.py:110} WARNING - /opt/***/dags/test.py:71: FutureWarning: Dropping of nuisance columns in DataFrame reductions (with 'numeric_only=None') is deprecated; in a future version this will raise TypeError.  Select only valid columns before calling the reduction.
  df.fillna(df.median(), inplace=True)

[2023-07-09T08:31:09.897+0000] {python.py:183} INFO - Done. Returned value was:    LocationAbbr          LocationDesc  ...       PCA1       PCA2
0            AR              Arkansas  ...   2.329192  -2.050249
1            CO              Colorado  ... -11.989688  -4.003571
2            DC  District of Columbia  ...  14.517821   7.995440
3            GA               Georgia  ...  10.997803  -0.242377
4            MI              Michigan  ...   5.155159  10.091829
6            OR                Oregon  ... -27.124871  -4.538535
9            WI             Wisconsin  ...   0.612669   7.753479
10           WI             Wisconsin  ...   0.612669   7.753479
11           AL               Alabama  ...   8.257815  -1.474515
12           ID                 Idaho  ... -21.480617  -2.968670
13           ID                 Idaho  ... -21.480617  -2.968670
14           IL              Illinois  ...   2.963120   4.525560
15           KS                Kansas  ...  -4.552090  -1.200352
16           KS                Kansas  ...  -4.552090  -1.200352
17           KS                Kansas  ...  -4.552090  -1.200352
18           LA             Louisiana  ...   3.583206  -5.253806

[16 rows x 36 columns]
[2023-07-09T08:31:09.977+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=apply_pca, execution_date=20230709T083057, start_date=20230709T083109, end_date=20230709T083109
[2023-07-09T08:31:10.033+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-09T08:31:10.069+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
